bioio_ome_zarr/tests/writers/test_ome_zarr_writer_V2.py

```diff
@@ -14,12 +14,65 @@
     OmeZarrWriterV2,
     chunk_size_from_memory_target,
     compute_level_chunk_sizes_zslice,
-    compute_level_shapes,
 )
 
 from ..conftest import array_constructor
 
 
```

Comment:
Not sure what to do with this. It's not as relevant as a util since we changed how level_shapes / scale work. I moved it into the test since that's the only place that uses it with the plan to remove it when the V2 is fully depreciated.